python/sparkdl/transformers/named_image.py

```diff
@@ -82,9 +88,11 @@ def getModelName(self):
         return self.getOrDefault(self.modelName)
 
     def _transform(self, dataset):
-        transformer = _NamedImageTransformer(inputCol=self.getInputCol(),
-                                             outputCol=self._getIntermediateOutputCol(),
-                                             modelName=self.getModelName(), featurize=False)
+        transformer = _NamedImageTransformer(
```

Comment:
one of the style things we're inconsistent about in this repo is how to handle wrapping function calls that span > 1 line. i think we'd decided to use multiple lines but try to fit as many arguments as possible onto each line, e.g:

```
        transformer = _NamedImageTransformer(
            inputCol=self.getInputCol(), outputCol=self._getIntermediateOutputCol(),
            modelName=self.getModelName(), featurize=False)
```
i'll let @sueann confirm, but i'm fine with this as is too :)

Reply:
I went with the path of least resistance while making changes, however I agree we should decide and document this change.

Reply:
Yeah I think it's better to compact the code as much as possible. It looks fine in this case but this pattern wastes a lot of space elsewhere and makes it harder to read in my opinion.